src/temp/Explore/CustomSwitch.tsx

```diff
@@ -46,7 +46,7 @@ export const StyledSwitch = styled(SmallSwitch)<SwitchProps>(
       color:
         (currentforestswitch && `${primaryColorNew}`) ||
         (restorationswitch && `${restorationToggleColorNew}`) ||
-        (deforestrationswitch && `${exploreRangeSliderColorNew}`),
+        (deforestrationswitch && `rgba(235, 87, 87)`),
```

Comment:
Instead of hard coding use this format : `${'rgba(var(--explore-range-background-new))'}` here.